rest-server/src/models/job.js

```diff
@@ -276,6 +280,63 @@ class Job {
       });
   }
 
+  getJobConfig(userName, jobName, next) {
```

Comment:
>getJobConfig [](start = 2, length = 12)

please add unit tests for getJobConfig and getJobSshInfo.

Reply:
Hi Fan. Done.